### DISCUSSION TITLE
add breadcrumbs to a theme

### DISCUSSION THREAD

**Discussion by vlzq** [GetPublii/Publii#1654] [Help]
Is there any way to add breadcrumbs to my custom theme? 

I haven't found any code snippet in the documentation to do something similar with the new page system.

**Comment by internettips** [comment]
The solution at: [https://cv.domagic.site/posts/breadcrumbs-for-publii](url) works.

**Reply by vlzq** [reply]
work in **post.hbs** not in **page.hbs**

![image](https://github.com/user-attachments/assets/6a1bf383-c3c2-43fb-aef4-713f0701ce8d)

**Comment by internettips** [comment]
@otono91 

For post breadcrumbs, as per the article in the previous link, try adding the following code to your CustomCSS in Publii:

`/* Add breadcrumbs */
.breadcrumbs {
  display: flex;
  justify-content: flex-start;
  list-style: none;
  margin: 0.5rem;
}

.breadcrumbs li {
  padding-right: 0.5rem;
}

.breadcrumbs li:nth-child(1) {
  padding-left: 0;
}`

For the new Pages feature in Publii [https://getpublii.com/blog/release-046.html](url), the breadcumbs method outlined above for Posts relies on tags. But the new Pages feature in Publii doesn't interact with tags -- Pages are just standalone pages with no relationship to other pages or posts (unless we specifically add discrete links). So this breadcrumb system won't work in the same way. 

It seems that any breadcumb system for Pages in Publii will need a different set-up (even though it displays okay for pages in Splitian theme, the functionality of course won't work as it is now since tags are not referenced in Pages). Maybe the Publii devs or someone else could produce a plugin that addresses this issue that covers both Pages and Posts in Publii. Fingers crossed.
